arlo-client/src/components/Audit/RoundManagement/index.tsx

```diff
@@ -91,7 +91,11 @@ const RoundManagement = ({ round, auditBoards, createAuditBoards }: IProps) => {
             >
               Download Audit Board Credentials for Data Entry
             </FormButton>
-            <RoundProgress />
+            <RoundProgress
+              round={round}
+              electionId={electionId}
+              jurisdictionId={jurisdictionId}
```

Comment:
instead of calling `useAuditBoards` within `RoundProgres`, if you pass the `auditBoards` from this parent component in as a prop `auditBoards={auditBoards}`, then they will stay up to date, since it's flowing from the same centralized state

Reply:
Aha, I missed that it was already in there. I looked for it, haha!